harvard-lts/etd-integration-tests
# Etd 181

**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/ETD-181

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
fixes the itest so it calls the dash service queue

# How should this be tested?

this is in dev. to test, run the itest:

https://ltsds-cloud-dev-1.lib.harvard.edu:10610/integration

and then check the rabbitmq web console to see that the message ends up in the etd_finished queue

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
